### DISCUSSION TITLE
Move https://github.com/datafuse-extras/clickhouse_driver into this repo?

### DISCUSSION THREAD

**Discussion by sundy-li** [databendlabs/opensrv#8] [Q&A]
cc @PsiACE

**Comment by PsiACE** [comment]
Make sense.

For now let's maintain opensrv-clickhouse and opensrv-mysql first. In the next phase, I will consider integrating clickhouse-* and maybe we will get opensrv-clickhouse-common, opensrv-clickhouse-dirver, and opensrv-clickhouse.
